src/System.Net.HttpListener/tests/HttpListenerRequestTests.cs

```diff
@@ -320,8 +328,7 @@ public static IEnumerable<object[]> ContentEncoding_TestData()
             });
         }
 
-        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotOneCoreUAP))]
-        [PlatformSpecific(TestPlatforms.Windows)]
+        [ConditionalFact(nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementation))]
```

Comment:
Issue number? Or a comment about why it's platform-specific?  I'll stop commenting on these.